lib/etc/openturns.conf.in

```diff
@@ -141,6 +141,14 @@
 
   <!-- OT::Cobyla parameters -->
   <Cobyla-DefaultRhoBeg value_float="0.1" />
+  
+  <!-- OT::Dlib parameters -->
+  <Dlib-DefaultWolfeRho                 value_float="0.01"    />
+  <Dlib-DefaultWolfeSigma               value_float ="0.9"    />
+  <Dlib-DefaultMaxSize                  value_int="10"        />
+  <Dlib-DefaultMaxLineSearchIterations  value_int="100"       />
+  <Dlib-DefaultInitialTrustRegionRadius value__float="1.0"    />
```

Comment:
value__float->double underscore here

Reply:
Please sort the keys by type, in alphabetical order within each type, and put the types in this order: bool before float before string before int.